style.css

```diff
@@ -0,0 +1,101 @@
+:root{
+--border-color:#d1d1d1;
+
+}
+
+
+.navbar{
+    position: sticky;
+    top: 0;
+    left: 0;
+    width: 100%;
+    z-index: 9;
+}
+
+.nav{
+    padding: 2px 2vw;
+    display: flex;
+    justify-content: space-between;
+}
+
+.nav-items{
+    display: flex;
+    align-items: center;
+}
+
+.brand-logo{
+    height:23rem;
+    width:23rem;
+}
+
+.search{
+    width: 600px;
+    display: flex;
+}
+
+.search-box{
+    width: 80%;
+    height: 40px;
+    padding: 10px;
+    border-top-left-radius: 10px;
+    border-bottom-left-radius: 10px;
+    border: 1px solid var(--border-color);
+    text-transform: capitalize;
+    background: none;
+    color: #a9a9a9;
+    outline: none;
```

Comment:
Instead of outline:none use 
 outline: 2px solid transparent;
 outline-offset: 2px;